containers/src/Kokkos_DynamicView.hpp

```diff
@@ -263,12 +263,17 @@ class DynamicView : public Kokkos::ViewTraits<DataType, P...> {
   unsigned m_chunk_size;  // 2 << (m_chunk_shift - 1)
 
  public:
-  //----------------------------------------------------------------------
-
+//----------------------------------------------------------------------
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
   /** \brief  Compatible view of array of scalar types */
-  using array_type =
+  using array_type KOKKOS_DEPRECATED_WITH_COMMENT(
+      "Use Kokkos_DynamicView::type insted.") =
+      DynamicView<typename traits::scalar_array_type,
+                  typename traits::device_type>;
+#endif
+  /** \brief  Compatible view of data type */
+  using type =
       DynamicView<typename traits::data_type, typename traits::device_type>;
```

Comment:
Same here.

Reply:
There is a use further down in L287

Reply:
Then you can declare `uniform_type` accordingly and void declaring a `type` alias.